bool front_left
bool middle_left
bool rear_left
bool front_right
bool middle_right
bool rear_right